src/libraries/Microsoft.Extensions.DependencyInjection/src/ServiceLookup/CallSiteFactory.cs

```diff
@@ -6,6 +6,7 @@
 using System.Collections.Generic;
 using System.Diagnostics;
 using System.Diagnostics.CodeAnalysis;
+using System.Linq;
```

Comment:
Will this cause System.Linq.Expressions to be included and not trimmable?

Reply:
If desired, this could easily be used. Looking at the rest of the project, `CallSiteChain` also has this using but it doesn't appear to be necessary. The other place it appears is `ExpressionResolverBuilder`, but perhaps that's fine because that whole class would be trimmed under this type of scenario?